.github/workflows/build_and_test.yml

```diff
@@ -206,7 +209,7 @@ jobs:
         run: |
           touch ${{ env.LOG_DIR}}/local-setup.log
           ./local-setup/launch.py local-setup/github-action-config.json 2>&1 | tee ${{ env.LOG_DIR}}/local-setup.log &
-          sleep 60
+          sleep 150
```

Comment:
Had to increase the time we wait until running scripts significantly. 60s was way too low, worker 2 was never really up for the M8 integration test (in M6 it was, because it itself has some sleeps as well). In the local setup alone we wait for 60s between starting workers, so after 60s worker 2 just started booting.